src/components/NavBar/NavBar.js

```diff
@@ -28,7 +28,7 @@ const NavBar = () => {
           <img className="bw-logo" alt="blue-witness-logo" src={bwlogo} />
         </Link>
         <a href="https://www.humanrightsfirst.org">
```

Comment:
Everything looks great! One small thing I would add would be target="_blank" rel="noopener noreferrer" inside the a tag. That way when anyone clicks on the link it will open in a new tab.